src/main/components/uml-element/assessable/assessable.tsx

```diff
@@ -151,10 +151,13 @@ export const assessable = (
           console.error('Could not parse artemis sgi', e);
           return;
         }
+        // TODO: following messages should be received from Artemis
+        const removeMessage = 'Do you want to remove the link to the assessment instruction?';
+        const tooltipMessage = 'Assessment Instruction: ' + instruction.instructionDescription;
```

Comment:
How are these translated later on? I think, I don't get it 😄

Reply:
actually, for now, translation is not working properly when you switch the language i Artemis you can still see the modeling editor in English. This is not part of this PR. In the future, we can send these messages from Artemis, I am not quite sure how the translation will work between Apollon and Artemis. So, I want to leave it as it is.